server/scripts/clubs.yml

```diff
@@ -0,0 +1,37 @@
+- name: Loolabs # All possible club fields listed
```

Comment:
I mean, most info should be public anyways so I don't see why not.

Reply:
I don't think there's any harm in pushing club data but we probably make a team-wide decision for all future population scripts. In the future, there might be data that must be private and hence can't be pushed to version control. It would then be confusing that some data is on github and some not.